File: packages/dd-trace/test/llmobs/parity/scenarios/openai-agents/tool-error.js
'use strict'

const { additionAgentWithToolErrors, setup } = require('./common')

async function main () {
  const { agents, finish } = setup()
  await agents.run(additionAgentWithToolErrors(), 'What is the sum of 1 and 2?')
  await finish()
}

main().catch(error => {
  process.stderr.write(`${error.stack ?? error}\n`)
  process.exitCode = 1
})

File: packages/dd-trace/test/llmobs/parity/scenarios/openai-agents/tool-error.py
import asyncio

from agents import Runner

from _agents_common import addition_agent_with_tool_errors
from _agents_common import finish
from _agents_common import setup


async def main():
    setup()
    await Runner.run(addition_agent_with_tool_errors(), "What is the sum of 1 and 2?")
    finish()


asyncio.run(main())
